### PR TITLE
Trackers: vlive.tv

### PR DESCRIPTION
**Pull request by ghost**
### URL: `https://www.vlive.tv/events`
Screenshot:
Screenshot

### Network Logs
network1
network2

### CODE REVIEW

**Comment by piquark6046** [comment]
1a2a5018760ee43d8e550907e4ad2aa08f6044c4
### URL: `https://www.vlive.tv/video/227059`
Screenshot: 
Screenshot

### Network Logs
network1
network2
network3

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -22,6 +22,10 @@
 ||nd.demdex.net/event?d_mid=$redirect=nooptext,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 !
+||vlive.tv/globalv-web/vam-web/old/vod/*/play
+||apis.naver.com/videostat/videostat/PlayTime.json
+||apis.naver.com/rmcnmv/rmcnmv/PlayCount.json
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:27]: > ||vlive.tv/globalv-web/vam-web/old/vod/*/play
||apis.naver.com/videostat/videostat/PlayTime.json
||apis.naver.com/rmcnmv/rmcnmv/PlayCount.json

Not sure about these. Better do not block them.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply SpywareFilter/sections/specific.txt:27]: Yep, I'm not sure about them too.

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt:27]: OK. I will change them.
Please wait.

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: Updated at 18025a6b428c0898ad8ca52dcdae50fc12ad5dcb

Comment (zloyden) [reply SpywareFilter/sections/specific.txt]: I guess it's ok now. One tip for future - if you add some changes, don't put them at the beggining.

**Review by zloyden — APPROVED** [review]

**Pull request merged**